static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
-                }
-              }
-            }
-          }
-        }
-      },
-      "patch": {
-        "summary": "Update prefetch configuration",
-        "tags": ["Prefetch Configurations"],
-        "description": "Updates the prefetch configuration associated with the origin.",
-        "requestBody": {
-          "content": {
-            "application/json": {
-              "schema": {
-                "type": "object",
-                "properties": {
-                  "cronSchedule": {
-                    "type": "string",
-                    "description": "Cron expression for scheduling."
-                  },
-                  "isRenderForMobile": {
-                    "type": "boolean",
-                    "description": "Whether to render for mobile devices or not."
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
                   }
                 }
               }
             }
           }
-        },
+        }
+      }
+    },
+    "/api/prerender/v1/render/{mobile}/{url}": {
+      "get": {
+        "summary": "Get prerendered page",
+        "tags": ["Prerender"],
+        "description": "Returns the HTML content of the rendered page.",
         "parameters": [
           {
-            "schema": { "type": "string", "minLength": 1 },
+            "schema": { "type": "boolean" },
             "in": "path",
-            "name": "origin",
+            "name": "mobile",
+            "required": true,
+            "description": "Whether to render as a mobile device or not."
+          },
+          {
+            "schema": { "type": "string" },
+            "in": "path",
+            "name": "url",
             "required": true,
-            "description": "Origin hostname"
+            "description": "URL to be rendered."
```

Comment:
```suggestion
            "description": "URL to render."
```

Reply:
Not sure about this one.  Did I change the meaning?

Reply:
Same meaning. I'll update it.